cmd/livepeer/starter/starter.go

```diff
@@ -526,16 +526,24 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 		}
 
 	} else {
-		var keystoreDir string
-		if _, err := os.Stat(*cfg.EthKeystorePath); !os.IsNotExist(err) {
-			keystoreDir, _ = filepath.Split(*cfg.EthKeystorePath)
+		var keystoreDir = filepath.Join(*cfg.Datadir, "keystore")
+		if keystoreInfo, err := ParseEthKeystorePath(*cfg.EthKeystorePath); err == nil {
+			if keystoreInfo.path != "" {
+				keystoreDir = keystoreInfo.path
+
+				if (keystoreInfo.address != ethcommon.Address{}) {
```

Comment:
```suggestion
			if keystoreInfo.path != "" {
				keystoreDir = keystoreInfo.path
			} else if (keystoreInfo.address != ethcommon.Address{}) {
```

It corresponds to my other comment that it's better to return either `path` or `address`.

Reply:
Completed